recipe/meta.yaml

```diff
@@ -23,9 +23,12 @@ source:
   sha256: e362e1fbf6b101ad71aa5314f21d00a4353b90efbe5cdedc5fcfb0f3b68cd791  # [win and cuda_major == "12"]
 
 build:
-  number: 0
+  number: 1
   # TODO: Enable Windows with CUDA 12 when it is supported in CI
   skip: True  # [win32 or osx or (win and cuda_major == "12")]
+  # Disable binary relocation to workaround patchelf issue
+  # xref: https://github.com/NixOS/patchelf/issues/492
+  binary_relocation: false
```

Comment:
Seeing this [in CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=715464&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=398 ):

```
ELF load command address/offset not properly aligned
```

Looks like issue ( https://github.com/NixOS/patchelf/issues/492 )

Disabling `binary_relocation` to workaround this issue